sdk/communication/azure-communication-identity/azure/communication/identity/_shared/user_credential_async.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-
+# pylint: disable=C4763
```

Comment:
In addition to disabling this, if we are going to use asyncio we should also support trio , see [here](https://github.com/Azure/azure-sdk-for-python/blob/d69024be7a3ecd92c375c8e2308e8fbd55441116/sdk/core/azure-core/azure/core/utils/_utils.py#L168-L188) an example of how azure-core did this

Reply:
Yes we can look into adding something to azure-core :)